### PR TITLE
Link new-ish md files and WWDC sessions

### PR DESCRIPTION
**Pull request by neonichu**
This should improve the discoverability of these resources.

### CODE REVIEW

**Comment by neonichu** [comment]
@swift-ci please smoke test

File: Documentation/README.md [hunk 1]
```diff
@@ -17,9 +17,11 @@ We’ve designed the system to make it really easy to share packages on services
     * [Dependency Hell](#dependency-hell)
```
Comment (MaxDesiatov) [inline Documentation/README.md:17]: Maybe we could also link to this `Documentation/README.md` from the repository-level `README.md`?

Comment (neonichu) [reply Documentation/README.md:17]: Good point, I think right now we link to the directory instead

**Comment by neonichu** [comment]
@swift-ci please smoke test

**Review by MaxDesiatov — APPROVED** [review]

**Review by abertelrud — APPROVED** [review]
Thanks!

**Comment by tomerd** [comment]
nice!

**Pull request merged**